generators/generator-base.js

```diff
@@ -1301,22 +1303,28 @@ module.exports = class extends PrivateBase {
     /**
      * get sorted list of entities according to changelog date (i.e. the order in which they were added)
      */
-    getExistingEntities() {
+    getExistingEntities(recreate = false) {
```

Comment:
Recreate? What's its use?

Reply:
core writes, directly to disk.
To make prettier work it needs to load the file and modify it on mem-fs.
this.fs.{exists|read} loads it, this.append sets it as modified.

maybe the var name has to be changed.